checks/fileparser/github_workflow.go

```diff
@@ -484,6 +484,16 @@ func IsPackagingWorkflow(workflow *actionlint.Workflow, fp string) (JobMatchResu
 			},
 			LogText: "candidate ruby publishing workflow using gem",
 		},
+		{
+			// Dart packages.
+			Steps: []*JobMatcherStep{
+				{
+					Run: "dart.*pub.*publish",
+				},
+			},
+			LogText: "candidate dart publishing workflow",
+		},
+
```

Comment:
is this meant to be in this PR?

Reply:
not really, but I slipped it it as I stumbled upon it reading the code